### PR TITLE
Adjust the request and limits (4096Mi RAM per 1000m CPU)

### PR DESCRIPTION
**Pull request by fredg02**

### CODE REVIEW

File: templates/jenkins/configuration.yml.hbs [hunk 1]
```diff
@@ -60,8 +60,8 @@ jenkins:
           name: "jnlp"
           resourceLimitCpu: "2375m"
```
Comment (mbarbero) [inline templates/jenkins/configuration.yml.hbs]: should be 2000

Comment (mbarbero) [reply templates/jenkins/configuration.yml.hbs]: also, 2375 would not work as it doesn't comply with limitrange.limits.container.max = 2000

Comment (fredg02) [reply templates/jenkins/configuration.yml.hbs]: Thanks, I missed that.

**Comment by mbarbero** [comment]
Please try a couple of build in dash and cbi before deploying more widely ;)

**Comment by fredg02** [comment]
Will do.

**Pull request merged**